core-bundle/tests/Controller/BackendPreviewSwitchControllerTest.php

```diff
@@ -170,17 +170,17 @@ private function mockRouter(): RouterInterface
     /**
      * @return TokenChecker&MockObject
      */
-    private function mockTokenChecker(?string $frontendUsername = null, bool $previewMode = true): TokenChecker
```

Comment:
Feels like the fact that these arguments are not used means the test is incomplete. I would suggest to revert that and create a ticket/PR to improve the test.

Reply:
> means the test is incomplete

No, it does not. It might as well be a left-over from an older version in which the arguments were used.

Reply:
well it obviously does not test what the controller does if there is a username or `isPreviewMode` is not `true`?

Reply:
Yep, looks like a test for `isPreviewMode = false` is missing, if that's a necessary case at all.

Reply:
Reverted in f1facf7dfe50eee127db070361108801325835ef.